Sources/WebSocketKit/PMCE.swift

```diff
@@ -717,11 +547,7 @@ extension PMCE.PMCEConfig: CustomDebugStringConvertible {
 
 extension PMCE.PMCEConfig: CustomStringConvertible {
     public var description: String {
-        """
-        PMCEConfig {
-          \(self.deflateConfig.debugDescription)
-        }
-        """
+       debugDescription
```

Comment:
If `description` just returns `debugDescription` then the latter is redundant. Put the `debugDescription` implementation here and remove the `CustomDebugStringConvertible` conformance entirely.